src/main/scala/org/broadinstitute/dsde/firecloud/utils/StreamingActor.scala

```diff
@@ -1,44 +0,0 @@
-package org.broadinstitute.dsde.firecloud.utils
-
-import akka.actor.{Actor, Props}
-import com.typesafe.scalalogging.LazyLogging
-import org.broadinstitute.dsde.firecloud.utils.StreamingActor._
-import spray.http.{ContentType, _}
-import spray.routing.RequestContext
-
-object StreamingActor {
-  case class FirstChunk(httpData: HttpData, remaining: Int)
-  case class NextChunk(httpData: HttpData, remaining: Int)
-  case class Ok(remaining: Int)
-  def props(ctx: RequestContext, contentType: ContentType, fileName: String) =
-    Props(new StreamingActor(ctx, contentType, fileName))
-}
-
-class StreamingActor(ctx: RequestContext, contentType: ContentType, fileName: String) extends Actor with LazyLogging {
```

Comment:
as noted in ExportEntitiesByTypeActor, I killed this actor off in favor of another approach